streamz/sources.py

```diff
@@ -71,3 +72,42 @@ def start(self):
                     yield last
                 else:
                     return
+
+
+class Kafka(Source):
+    """ Accepts messages from Kafka
+
+    Parameters
+    ----------
+    topics: list of str
+        Labels of Kafka topics to consume from
+    url: str
+        Connection string (host:port) by which to reach Kafka
+    group: str
+        Identity of the consumer. If multiple sources share the same group,
+        each message will be passed to only one of them.
+    poll_interval: number
+        Seconds that the thread sleeps between polling Kafka for new messages
+    """
+    def __init__(self, topics, url='localhost:9092', group='streamz',
+                 poll_interval=0.1):
```

Comment:
I suggest that we just pass through the entire dictionary argument to `ck.Consumer`.  I'm not personally confident enough about Kafka that I would want to in any way control how users send keywords to the kafka client.